files/etc/profile.d/workarea.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Configure all login shells to use the settings in `/etc/workarea.env`.
+# Set up the Workarea application's configuration if this is the first
+# time someone has logged in.
+#
+
+set -a
```

Comment:
this `set -a` and `set +a` stuff around sourcing the .env file causes all the variables defined in the file to be exported, allowing them to be used in the shell session as well as in systemd. nice little hack thanks in part to the #systemd IRC channel :)